packages/@react-stately/select/src/useSelectState.ts

```diff
@@ -15,14 +15,16 @@ import {SelectProps} from '@react-types/select';
 import {SingleSelectListState, useSingleSelectListState} from '@react-stately/list';
 import {useState} from 'react';
 
-export interface SelectState<T> extends SingleSelectListState<T>, MenuTriggerState {
+interface SelectStateTmp<T> extends SingleSelectListState<T>, MenuTriggerState {
```

Comment:
we can make this one line and delete the other one I think
```suggestion
export interface SelectState<T> extends Omit<SingleSelectListState<T>, 'disallowEmptySelection'>, MenuTriggerState {
```

Reply:
Great idea, thanks!